### ISSUE TITLE
dotgit restore error: `unable to find "xxx" in repo, skipping`, but the file is in the repo

### ISSUE THREAD

**Issue by montaropdf** [kobus-v-schoor/dotgit#29]
Hello,

I have just installed and tried dotgit on a new machine.

Even if the files are present in the repository, dotgit restore, will generate the message `unable to find "xxx" in repo, skipping`, for every file in `filelist`.

Below is an exerpt from my `filelist`.
```
# Groups
singularity=desktop,gaming,private,singularity
tanko=laptop,org,private,tanko

# Common files
.emacs
.config/emacs/core/profile.el
.config/emacs/core/init-theme.el

# Configuration for the host singularity
.config/emacs/singularity/profile.el:singularity
```

There is no files, nor directories, nor links at the target location in my home dir.

I use a Fedora 33 Linux and I have installed dotgit with `pip3 --user`.

**Comment by montaropdf** [comment]
I have found the source of the error.

In fact, 1) I didn't perform a `dotgit init` to begin with and 2) I didn't copy files from my home folder to my dotfiles repo using `dotgit update`, but I was creating brand new files and directories in the root tree of the repository. Finally, I didn't notice the particular directory tree of your dotgit example, I have seen nowhere, or didn't notice, in the documentation that categories where translated to subtrees in `<repo>/dotfiles/`

After moving the files from my repo to their intended target location and run a `dotgit update`, I got a correct tree of directories and files in my dotfiles repository and symlink in my home folder.

May I ask for an update to the documentation, to clearly state what will be the directory structure of the repo after `dotgit init` and `dotgit update`?
